10_Examples/will_onboarding1/include/will_onboarding1/Mather.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <chrono>
+#include <memory>
+#include <string>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/int32.hpp"
+
+using namespace std::chrono_literals;
+using namespace std;
+using std::placeholders::_1;
+
+namespace Math_Space {
+
+class Mather : public rclcpp::Node {
+public:
+	Mather(int a, int b);
+
+	int decode(int encoded);
+
+	void topic_callback(const std_msgs::msg::Int32::SharedPtr msg);
+
+	void timer_callback();
```

Comment:
Header files are sometimes the only thing people can count on when using an external library, so this is where you would put comment blocks above every method describing what it does, if it throws errors, what edge cases, etc.

Example:
![image](https://github.com/VEXU-GHOST/VEXU_GHOST/assets/47650195/a7343555-cf31-49fd-9515-4bd2ed1b82c0)